apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/McsClaimTransformerV2.java

```diff
@@ -220,16 +224,22 @@ private List<Claim.ItemComponent> getItems(RdaMcsClaim claimGroup) {
               Claim.ItemComponent item;
 
               if (Strings.isNotBlank(detail.getIdrProcCode())) {
+                List<Coding> codings = new ArrayList<>();
```

Comment:
For my own knowledge, what's the reason for this change? Seems we're not using the `createCodeableConcept` function anymore

Reply:
@aschey-forpeople Good question! So this is still achieving the same purpose except it now allows multiple codings to exist for a productOrService element. Multiple codes, specifically idr_proc_code and idr_dtl_ndc, can exist on the the same idr detail number and to avoid them overwriting each other, i allowed multiple codings to exist for those two codes. They can exist within the same CodeableConcept since it can have multiple codings per FHIR specs. Here's a slack thread where Alex better explains the reasoning: https://cmsgov.slack.com/archives/C01QZ221E20/p1713887028388529?thread_ts=1713285885.768199&cid=C01QZ221E20

Reply:
Interesting, thanks!